### PR TITLE
Interop

### PR DESCRIPTION
**Pull request by reqshark**
we've added the interop messaging layer for blackrock and goldman sachs

### CODE REVIEW

File: minuet/symphony/Paragon.Plugins.MessagBus/App.config [hunk 1]
```diff
@@ -1,15 +1,11 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
   <appSettings>
-    <add key="BrokerLibraryPath" value="\\firmwide\appstore\1188\Test\messagebus.jar"/>
-    <add key="BrokerExePath" value="detect"/>
+    <add key="BrokerLibraryPath" value="C:\Users\Stamatakis\Documents\GitHub\SFE-Minuet-DesktopClient\minuet\messagebus\target\messagebus-1.0.0-SNAPSHOT.jar"/>
```
Comment (lneir) [inline minuet/symphony/Paragon.Plugins.MessagBus/App.config:4]: what are these paths?  ref to stamatakis?

File: minuet/symphony/Paragon.Plugins.MessagBus/App.config [hunk 2]
```diff
@@ -1,15 +1,11 @@
...
     <add key="BrokerLoggingConfigFile" value="detect"/>
     -->
   </appSettings>
-  <startup>
-    <supportedRuntime version="v4.0"/>
-    <supportedRuntime version="v2.0.50727"/>
-  </startup>
-</configuration>
\ No newline at end of file
+</configuration>
```
Comment (lneir) [inline minuet/symphony/Paragon.Plugins.MessagBus/App.config:15]: merge here lost below, this is needed to ensure code run with .net3.5 or higher.
<startup>
  <supportedRuntime version="v4.0"/>
  <supportedRuntime version="v2.0.50727"/>
</startup>

File: minuet/symphony/Symphony/Symphony/SymphonyInterop.js [hunk 3]
```diff
@@ -1,23 +1,48 @@
...
+
+    mb.subscribe("com.symphony.api", function (envelope) {
+        console.log("com.symphony.api/incoming envelope: ", envelope);
+
+        window.postMessage(envelope.message, "*");
+
+    });
+
+    mb.subscribe("com.symphony.system.interop", function (envelope) {
+        console.log("system.symphony.com/message: ", envelope);
```
Comment (lneir) [inline minuet/symphony/Symphony/Symphony/SymphonyInterop.js:14]: why all the console.logs - any impact on perf.  needed?

File: minuet/symphony/Symphony/Symphony/SymphonyInterop.js [hunk 4]
```diff
@@ -1,23 +1,48 @@
...
-        
-        window.postMessage({
-            "version": 1,
-            "event": "open:im",
-            "payload": {
-                "userName": message.message.userName,
-                "sendMessage": message.message.content
+var symphony = symphony || {};
+symphony.interop = {createChannel: function(){
+    var mb = meow.messagebroker();
```
Comment (lneir) [inline minuet/symphony/Symphony/Symphony/SymphonyInterop.js:3]: how do we get access to meow?  is this a global?

File: minuet/symphony/Symphony/Symphony/config.json [hunk 5]
```diff
@@ -1,3 +1,5 @@
-﻿{
-  "url" : "https://beta.symphony.com"
+﻿{
+  "url" : "https://beta.symphony.com",
```
Comment (lneir) [inline minuet/symphony/Symphony/Symphony/config.json]: what are url2 and url3 used for?  testing?  if so, comment out.

File: minuet/symphony/Paragon.Plugins.MessagBus/App.config (hunk 1 above)
Comment (reqshark) [inline minuet/symphony/Paragon.Plugins.MessagBus/App.config:4]: for running and testing the message broker during local development.

we can probably just leave @stamaw's path there. Besides, @matthewgardner and I swap it out manually when testing the inbound/outbound messaging service.

for dev connivence we ought to figure out some auto-detect when Visual Studio loads up.

File: minuet/symphony/Symphony/Symphony/SymphonyInterop.js (hunk 4 above)
Comment (reqshark) [inline minuet/symphony/Symphony/Symphony/SymphonyInterop.js:3]: `meow.js` gets listed before `SymphonyInterop.js` in the manifest, apparently it gets loaded first.

File: minuet/symphony/Symphony/Symphony/SymphonyInterop.js (hunk 3 above)
Comment (reqshark) [inline minuet/symphony/Symphony/Symphony/SymphonyInterop.js:14]: if you're asking what is the slowest thing in JavaScript? the answer is:

``` js
console.log('something or other that is printing during production')
```

anyway, this is something that probably helps all sides during dev, to observe UI Hook payloads in transport.

File: minuet/symphony/Symphony/Symphony/config.json (hunk 5 above)
Comment (reqshark) [inline minuet/symphony/Symphony/Symphony/config.json]: not sure how strict the interpreter here is, but commenting is not valid JSON, i'll remove the extra urls.

**Comment by lneir** [comment]
looks good :+1:

**Pull request merged**